boards/common/gd32v/include/cfg_timer_default.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2020 Koen Zandberg <koen@bergzand.net>
+ *               2023 Gunar Schorcht <gunar@schorcht.net>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_gd32v
+ * @{
+ *
+ * @file
+ * @brief       Default timer configuration for GD32VF103 boards
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#ifndef CFG_TIMER_DEFAULT_H
+#define CFG_TIMER_DEFAULT_H
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name   Timer configuration
+ *
+ * All GD32VF103xx variants have at least one advanced timer TIMER0 and two
+ * general timers TIMER1 and TIMER2. GD32VF10x8 and GD32VF10xB have two
+ * additional general timers TIMER3 and TIMER4.
+ *
+ * @{
+ */
+static const timer_conf_t timer_config[] = {
+    {
+        .dev      = TIMER1,
+        .max      = 0x0000ffff,
+        .rcu_mask = RCU_APB1EN_TIMER1EN_Msk,
+        .bus      = APB1,
+        .irqn     = TIMER1_IRQn
+    },
+    {
+        .dev      = TIMER2,
+        .max      = 0x0000ffff,
+        .rcu_mask = RCU_APB1EN_TIMER2EN_Msk,
+        .bus      = APB1,
+        .irqn     = TIMER2_IRQn
+    }
+#if defined(GD32VF103C8) || defined(GD32VF103CB) || \
+    defined(GD32VF103R8) || defined(GD32VF103RB) || \
+    defined(GD32VF103T8) || defined(GD32VF103TB) || \
+    defined(GD32VF103V8) || defined(GD32VF103VB)
```

Comment:
Unfortunatly not, since there are no separate definitions for the MCU models. That is RCU_APB1EN_TIMER3EN_Msk/RCU_APB1EN_TIMER4EN_Msk

Ooops, there are no such definitions. I should test variable `CPU_MODEL` or define these identifiers as macros.

Reply:
Fixed.